tests/templates/kuttl/orphaned-resources/03-update-druid.yaml.j2

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+metadata:
+  name: install-druid
+timeout: 600
+---
+apiVersion: druid.stackable.tech/v1alpha1
+kind: DruidCluster
+metadata:
+  name: derby-druid
+spec:
+  version: {{ test_scenario['values']['druid'] }}
+  zookeeperConfigMapName: druid-znode
+  metadataStorageDatabase:
+    dbType: derby
+    connString: jdbc:derby://localhost:1527/var/druid/metadata.db;create=true
+    host: localhost
+    port: 1527
+  deepStorage:
+    hdfs:
+      configMapName: druid-hdfs
+      directory: /druid
+  brokers:
+    roleGroups:
+      default: null
+      newrg:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        config: {}
```

Comment:
Can be removed (in the other roles as well)

Reply:
Done.